src/styles/blocks/page.scss

```diff
@@ -0,0 +1,10 @@
+html {
+  font-family: Roboto, serif;
```

Comment:
The font-family declaration should include a generic family name as a fallback. Consider using 'sans-serif' instead of 'serif' to match the style of 'Roboto'.